lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -69,18 +67,16 @@
  ticklabel ha           right  center   right  center
  axislabel ha           right  center   right  center
 
-
 Ticks are by default direct opposite side of the ticklabels. To make
```

Comment:
Wording is a bit strange, but never mind. I think the content is not true for the new style anymore. But that's probably for another PR...